code/__DEFINES/tools.dm

```diff
@@ -18,6 +18,7 @@
 #define TOOL_KNIFE			"knife"
 #define TOOL_BLOODFILTER	"bloodfilter"
 #define TOOL_ROLLINGPIN		"rollingpin"
+#define TOOL_SAFETYKIT		"safetykit"
```

Comment:
I am fairly sure you don't need it, just do istype check in the regenerator code.
Tool types are only really used when several unrelated objects can do same thing(I.e. knives being used as screwdriver).